### PR TITLE
ocrmypdf 6.1.2

### PR DESCRIPTION
**Pull request by jbarlow83**
Manual update to add new dependency on MuPDF/PyMuPDF.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

* `mupdf-tools` does not generate a dynamic library, only a static one, `libmupdf.a`, so I declared this a `:build` dependency. However, this detail could be dropped.

* The maintainer of PyMuPDF only delivers Python wheels for v1.12.4 on PyPI, so that resource is taken from GitHub. Future versions of PyMuPDF should be available on PyPI.

### CODE REVIEW

File: Formula/ocrmypdf.rb [hunk 1]
```diff
@@ -3,16 +3,10 @@ class Ocrmypdf < Formula
 
   desc "Adds an OCR text layer to scanned PDF files"
   homepage "https://github.com/jbarlow83/OCRmyPDF"
-  url "https://files.pythonhosted.org/packages/cc/b0/6d037507a945c5941188ce5fac17dcd0f68fde729664279aec5a2e84bc21/ocrmypdf-5.7.0.tar.gz"
-  sha256 "4f239823b266d6268eb54f62d439943ee2be70d9989c32855e69e9a610136ff8"
-
-  bottle do
```
Comment (commitay) [inline Formula/ocrmypdf.rb]: Please restore the bottle block, it should not be edited or removed manually.

Comment (jbarlow83) [reply Formula/ocrmypdf.rb]: Restored and squashed.

**Comment by ilovezfs** [comment]
Is this dependency required?

**Comment by jbarlow83** [comment]
@ilovezfs In 6.0.0 PyMuPDF was required. It is now an optional but recommended dependency. I changed it to optional because Debian/Ubuntu do not currently have a package for PyMuPDF. The reason there is no package currently is that MuPDF only generates a static library and must be recompiled with `-fPIC` to link with PyMuPDF. On macOS, however, libmupdf.a from MuPDF can be linked without recompiling.

ocrmypdf is tested [with and without PyMuPDF](https://travis-ci.org/jbarlow83/OCRmyPDF/builds/360438176) in continuous integration (EXTRAS=[fitz], the library’s internal name). Without PyMuPDF, the most noticeable difference is that OCRmyPDF is slower, and the output file size is much larger in some use cases.

(defusedxml, also added recently, is required but pure Python so should pose no difficulties.)

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @jbarlow83!

**Pull request closed without merging**